operator/operator_test.go

```diff
@@ -178,7 +183,9 @@ func (e *environment) WaitPhase() {
 
 func (e *environment) Shutdown() {
 	if e.cmd != nil {
-		e.cmd.Process.Kill()
+		if err := e.cmd.Process.Kill(); err != nil {
+			log.Warn("Could not kill process:", err)
+		}
```

Comment:
you could use `e.t` to assert that this doesn't fail.

Reply:
This will be executed in the test cleanup, so i dont think we should/need to let it fail here.